tox.ini

```diff
@@ -23,3 +23,6 @@ commands =
 [testenv:list-all]
 commands =
     m8s-list-containers
+
+[testenv:venv]
+commands = {posargs}
```

Comment:
Without this some "fatter" applications will not start, as stuff like `TERM` or `LANG` are not set.
```suggestion
passenv = *
commands = {posargs}
```

Reply:
Updated the code and commit message.